cartesia/voices.py

```diff
@@ -117,6 +118,55 @@ def create(
 
         return response.json()
 
+    def create_hifi_clone(
```

Comment:
```suggestion
    def hifi_clone
```

already a verb so this is cleaner